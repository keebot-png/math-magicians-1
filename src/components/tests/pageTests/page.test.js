import { render, screen } from '@testing-library/react';
import HomePage from '../../homePage/home';
import Calculator from '../../calculatorPage/calculator';
import Quotes from '../../quotePage/quote';

describe('Test React Page Components', () => {
  test('Test Home Page is rendered', () => {
    render(<HomePage />);

    expect(screen.getAllByText('Welcome to our page!')).toMatchSnapshot();
  });

  test('Test Calculator Page is rendered', () => {
    render(<Calculator />);

    expect(screen.findAllByTitle('Lets do some math!')).toMatchSnapshot();
  });

  test('Test Quote Page is rendered', () => {
    render(<Quotes />);

    expect(screen.findAllByTitle('Math Quote')).toMatchSnapshot();
  });
});
