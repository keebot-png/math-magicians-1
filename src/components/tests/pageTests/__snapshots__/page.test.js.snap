// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test React Page Components Test Calculator Page is rendered 1`] = `Promise {}`;

exports[`Test React Page Components Test Home Page is rendered 1`] = `
Array [
  <h1
    class="title"
  >
    Welcome to our page!
  </h1>,
]
`;

exports[`Test React Page Components Test Quote Page is rendered 1`] = `Promise {}`;
